scalameta/parsers/shared/src/main/scala/scala/meta/internal/parsers/LazyTokenIterator.scala

```diff
@@ -155,43 +140,6 @@ private[parsers] class LazyTokenIterator private (
       case _ => false
     })
 
-  private def mkIndentToken(pointPos: Int): Token = {
```

Comment:
We don't use these aside from LazyTokenIterator, why does it make sense to move them?

Reply:
everything will move, in steps